userspace/sysdig/sysdig.h

```diff
@@ -20,6 +20,10 @@ limitations under the License.
 #pragma once
 
 #include <config_sysdig.h>
+#if defined(HAS_CAPTURE) && !defined(_WIN32)
+#include "driver_config.h"
+#endif // HAS_CAPTURE
+
```

Comment:
Can we drop this? Even having a concept of a driver is libscap's internal detail.